datastore/src/main/kotlin/com/openlattice/organizations/controllers/OrganizationsController.kt

```diff
@@ -109,15 +109,10 @@ class OrganizationsController : AuthorizingComponent, OrganizationsApi {
     @PostMapping(value = ["", "/"], consumes = [MediaType.APPLICATION_JSON_VALUE])
     override fun createOrganizationIfNotExists(@RequestBody organization: Organization): UUID {
         Preconditions.checkArgument(
-                organization.connections.isEmpty() || isAdmin(),
+                organization.connections.isEmpty() || isAdmin,
                 "Must be admin to specify auto-enrollments"
         )
-        organizations.createOrganization(Principals.getCurrentUser(), organization)
-        securableObjectTypes.createSecurableObjectType(
-                AclKey(organization.id),
-                SecurableObjectType.Organization
-        )
```

Comment:
Why does this not need to exist anymore?

Reply:
it never did. this shouldn't even be in the controller anyway.